lib/db-admin.js

```diff
@@ -0,0 +1,41 @@
+import { compareDesc, parseISO } from 'date-fns';
+
+import firebase from './firebase-admin';
+
+export async function getAllFeedback(siteId) {
+  try {
+    const snapshot = await firebase
+      .collection('feedback')
+      .where('siteId', '==', siteId)
+      .get();
```

Comment:
Btw, I believe if you use the `Timestamp` type for the `createdAt` field in Firestore, you can do`orderBy('createdAt', 'desc')` here. 
This should let you leverage the default Firestore indexing on the `createdAt` field and avoid sorting the full in-memory list below.

Reply:
I looked into this a bit when initially researching how to handle dates with Firebase. The [Timestamp](https://firebase.google.com/docs/reference/android/com/google/firebase/Timestamp) classes uses milliseconds since epoch, which I've never really been a fan of. 

Both are completely valid use cases (`Timestamp` or formatted ISO strings). It mostly comes down to personal preference. I like the fact that ISO strings are human-readable. It's possible that large data sets would have an issue sorting again, but I don't think that will be an issue with this application. 

https://stackoverflow.com/questions/10286204/the-right-json-date-format

Appreciate the insightful comment!

Reply:
I was trying with `Timestamp` and it is a bit cumbersome to  spread the data from doc since firebase Timestamp is an object.
Is this also one of the reason you use ISO string?

Reply:
Good point. That wasn't one of my original intentions, but I could see how that would be annoying.